app/assets/stylesheets/_metrics_filter_panel.scss

```diff
@@ -70,21 +95,30 @@
 }
 
 .m-search-filter {
-  display: flex;
-
-  label {
-    display: none;
-  }
+  display: block;
+  position: relative;
+  flex: 1 0 0;
 
   .a-search-input {
-    flex-grow: 1;
     margin-right: 0;
-    z-index: 1;
+    z-index: 0;
+
+    /* @include core-16; */
+    /* @include box-sizing(border-box); */
+    position: relative;
+    width: 86%;
+    width: calc(100% - 36px);
```

Comment:
Yeah, [there's a mixin in the frontend toolkit](https://github.com/alphagov/govuk_frontend_toolkit/blob/e03e7f56e1b70194310ee970a3ccf00bd9376b36/stylesheets/_css3.scss#L81) but it's not really much better than this by itself.